data/accounts/store.go

```diff
@@ -46,6 +46,7 @@ func secureUser(user *User) *User {
 	return user
 }
 
+// nolint : dupl
 func (s *Store) getUser(ctx context.Context, name string) (user *User, err error) {
 	if user, err = s.rawUser(ctx, name); err != nil {
```

Comment:
I know what named return values are for.  ;)  Converting `=` to `:=` handles declaration. My point is about consistency, as well as using named return values in a way that doesn't really increase comprehension. Not to bikeshed the issue, but we should be more consistent.